spec/nameable/latin_spec.rb

```diff
@@ -1,284 +1,286 @@
-    it prefix do
-      expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Ms.')
+    %w{Ms Ms. Miss}.each do |prefix|
+      it prefix do
+        expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Ms.')
+      end
     end
-  end
 
-  %w{Dr Dr. Doctor}.each do |prefix|
-    it prefix do
-      expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Dr.')
+    %w{Dr Dr. Doctor}.each do |prefix|
+      it prefix do
+        expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Dr.')
+      end
     end
-  end
 
-  %w{Rev Rev. Reverand}.each do |prefix|
-    it prefix do
-      expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Rev.')
+    %w{Rev Rev. Reverand}.each do |prefix|
+      it prefix do
+        expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Rev.')
+      end
     end
-  end
 
-  %w{Fr Fr. Friar}.each do |prefix|
-    it prefix do
-      expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Fr.')
+    %w{Fr Fr. Friar}.each do |prefix|
+      it prefix do
+        expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Fr.')
+      end
     end
-  end
 
-  %w{Master Sir}.each do |prefix|
-    it prefix do
-      expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq(prefix)
+    %w{Master Sir}.each do |prefix|
+      it prefix do
+        expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq(prefix)
+      end
     end
+
   end
 
-end
+  context('suffix patterns (formatted)') do
+    %w{Sr Sr. Senior}.each do |suffix|
+      it suffix do
+        expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq('Sr.')
+      end
+    end
 
-context('suffix patterns (formatted)') do
-  %w{Sr Sr. Senior}.each do |suffix|
-    it suffix do
-      expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq('Sr.')
+    %w{Jr Jr. Junior}.each do |suffix|
+      it suffix do
+        expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq('Jr.')
+      end
     end
-  end
 
-  %w{Jr Jr. Junior}.each do |suffix|
-    it suffix do
-      expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq('Jr.')
+    %w{Esq Esq. Esquire}.each do |suffix|
+      it suffix do
+        expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq('Esq.')
+      end
     end
-  end
 
-  %w{Esq Esq. Esquire}.each do |suffix|
-    it suffix do
-      expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq('Esq.')
+    %w{PHD PhD Ph.D Ph.D. P.H.D.}.each do |suffix|
+      it suffix do
+        expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq('Ph.D.')
+      end
     end
+
   end
 
-  %w{PHD PhD Ph.D Ph.D. P.H.D.}.each do |suffix|
-    it suffix do
-      expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq('Ph.D.')
+  context('suffix patterns (generational)') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.